client.go

```diff
@@ -386,6 +390,24 @@ func (s *Client) LoopOut(globalCtx context.Context,
 	}, nil
 }
 
+// getExpiry returns an absolute expiry height based on the sweep confirmation
+// target, constrained by the server terms.
+func (s *Client) getExpiry(height int32, terms *LoopOutTerms,
+	confTarget int32) (int32, error) {
+
+	switch {
+	case confTarget < terms.MinCltvDelta:
+		return terms.MinCltvDelta, nil
```

Comment:
height + MinCltvDelta?

Reply:
Ah, that is why I didn't do the early retuns I remember now.

Reply:
Fixed